### PR TITLE
[Translation zh-tw] Tools > Lighthouse (1)

### PR DESCRIPTION
**Pull request by limhenry**
Chinese (Traditional) translation of:
- src/content/zh-tw/tools/lighthouse/audits/_template.md
- src/content/zh-tw/tools/lighthouse/audits/appcache.md
- src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md
- src/content/zh-tw/tools/lighthouse/audits/cache-contains-start_url.md
- src/content/zh-tw/tools/lighthouse/audits/console-time.md

Note: All the articles are directly translated from zh-CN.

### CODE REVIEW

**Comment by limhenry** [comment]
@petele looks like all the articles in `src/content/zh-tw/tools/lighthouse/audits` are missing `wf_blink_components` (which is not exists in zh-cn), do I need to add `wf_blink_components` to all the articles?

**Comment by petele** [comment]
If it's not in the original EN source, you can add `{# wf_blink_components: N/A #}`

**Comment by limhenry** [comment]
OK, then don't merge [Translation zh-tw] Tools > Lighthouse (1 - 8) first. I will update that. :)

**Comment by petele** [comment]
Actually - i've updated the tests so that `wf_blink_components` are not required on translated files. If these are good, lmk and I'll merge

**Comment by limhenry** [comment]
cool! then you may proceed to merge to PR 😄

File: src/content/zh-tw/tools/lighthouse/audits/appcache.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description:“網站不使用應用緩存”Lighthouse 審查的參考文檔。
+
+{# wf_updated_on: 2017-01-04 #}
+{# wf_published_on: 2017-01-04 #}
+
+# 網站不使用應用緩存 {: .page-title }
+
+## 爲什麼說此審查非常重要{: #why }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/appcache.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/appcache.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+應用緩存（也稱爲 AppCache）已[棄用][deprecated]。
+
+[deprecated]: https://html.spec.whatwg.org/multipage/browsers.html#offline
+
+## 如何通過此審查{: #how }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/appcache.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description:“網站不使用延遲首次繪製的鏈接標記”與“網站在標頭中不使用延遲首次繪製的腳本標記”Lighthouse 審查的參考文檔。
+
+{# wf_updated_on: 2016-12-01 #}
+{# wf_published_on: 2016-12-01 #}
+
+# 網站不使用延遲首次繪製的資源 {: .page-title }
+
+## 爲什麼說此審查非常重要{: #why }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+快速的頁面加載可提高對用戶的吸引力、增加網頁瀏覽量和提高轉化率。
+
+
+通過內聯首次繪製所需的鏈接和腳本，並延遲首次繪製不需要的鏈接和腳本，您可以提升頁面加載速度。
+
+
+## 如何通過此審查{: #how }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+在您的報告中，Lighthouse 列出了其檢測到的所有阻塞渲染的鏈接或腳本。
+您的目標是減少這些鏈接或腳本的數量。
+
+正如[如何實現審查](#implementation)中所述，Lighthouse 標記三種類型的阻塞渲染的鏈接：腳本、樣式表和 HTML 導入。如何進行優化取決於您正在使用的資源類型。
+
+注：如果某個資源稱爲“關鍵資源”，則意味着首次繪製需要該資源或該資源對頁面的核心功能至關重要。
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/blocking-resources.md]: `注:` should not be translated, the rest of the text should

File: src/content/zh-tw/tools/lighthouse/audits/cache-contains-start_url.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description:“緩存包含來自清單的 start_url”Lighthouse 審查的參考文檔。
+
+{# wf_updated_on: 2016-09-15 #}
+{# wf_published_on: 2016-09-15 #}
+
+# 緩存包含來自清單的 start_url {: .page-title }
+
+## 爲什麼說此審查非常重要{: #why }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/cache-contains-start_url.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/cache-contains-start_url.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+可確保在離線狀態下從移動設備主屏幕正確啓動 Progressive Web App。
+
+
+## 如何通過此審查{: #how }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/cache-contains-start_url.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/console-time.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description:“網站在其自身的腳本中不使用 console.time()”Lighthouse 審查的參考文檔。
+
+{# wf_updated_on: 2016-12-01 #}
+{# wf_published_on: 2016-12-01 #}
+
+# 網站在其自身的腳本中不使用 console.time() {: .page-title }
+
+## 爲什麼說此審查非常重要{: #why }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/console-time.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/console-time.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+* 與 Chrome DevTools Timeline 相集成。在 Timeline 錄製期間調用 User Timing 函數 `performance.measure()` 時，DevTools 自動將此測量結果添加到 Timeline 的結果中，如以下屏幕截圖中的 `my custom measurement` 標籤中所示。
+
+
+
+
+![Chrome DevTools Timeline 中的 User Timing 測量結果][timeline]
+
+[timeline]: /web/tools/lighthouse/images/user-timing-measurement-in-devtools.png
+
+## 如何通過此審查{: #how }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/console-time.md]: should be a space after the text and before the `{:`

File: src/content/zh-tw/tools/lighthouse/audits/_template.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description:“AUDIT_NAME”Lighthouse 審查的參考文檔。
+
+{# wf_updated_on: 2016-10-04 #}
+{# wf_published_on: 2016-10-04 #}
+
+# AUDIT_NAME {: .page-title }
+
+## 爲什麼說此審查非常重要{: #why }
```
Comment (petele) [inline src/content/zh-tw/tools/lighthouse/audits/_template.md]: looks like there's two more here, sorry!

**Pull request merged**